rules/crowdstrike_rules/crowdstrike_base64_encoded_args.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from panther_base_helpers import crowdstrike_process_alert_context
+
+
+def rule(event):
+    # List of command line tools to monitor for execution with Base64 encoded arguments
+    command_line_tools = {
+        "powershell.exe",
+        "cmd.exe",
+        "cscript.exe",
+        "wscript.exe",
+        "rundll32.exe",
+    }
+
+    # Define a regular expression pattern to match Base64 encoded strings
+    base64_pattern = re.compile(r"[A-Za-z0-9+/]{10,}[=]{0,2}")
```

Comment:
I suspect this wouldn't be effective in detecting base64 since any alphanumeric string of length >10 would match this

Reply:
I'd try using the pattern used in the standard_dns detection 
base64_pattern = re.compile(
        r"^(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?\."
    )
   and add a test case to check if it would trigger on normal strings

Reply:
Good call out. Updated the regex to be very similar to the standard dns detection with adding different anchors. I also updated the logic to split at space and check each `arg` of the list to properly anchor.